docs/eog/case-studies-cw20-timeline.rst

```diff
@@ -0,0 +1,147 @@
+  - Planned break-out groups distribution (sign up sheet for Discussion Groups session, randomly assigned for Collaborative Ideas session).
+  - Planned for video recording infrastructure (easy recording of sessions via Zoom).
+  - Had a Code of Conduct committee pre-event meeting.
+  - Constructed feedback form draft.
+
+
+Month 0
+^^^^^^^^^^^^^^^^^^^^^^
+
+Ran event!
+
+
+Month 1 (April 2020)
+^^^^^^^^^^^^^^^^^^^^^^
+
+Agenda
+  - Updated the agenda with links to resources.
+  
+Feedback
+  - Emailed participants with a feedback form within a week after the event.
+  - Sent a reminder email a few days later.
+  - Held a feedback prize draw to thank them for taking the time.
+  
+Publicity
+  - Published a `blog post with highlights <https://software.ac.uk/blog/2020-04-21-highlights-collaborations-workshop-2020>`_ from the event.
+  - Published a `participant's experience of CW20 <https://software.ac.uk/blog/2020-04-29-ssi-collaborations-workshop-2020-remote-unconference-experience-and-notes>`_.
+  
+Resources
+  - Updated language of event website to show that it was in the past.
+  - Created a `Figshare portal <https://cw20.figshare.com/>`_ to share CW20 outputs
+  - Chased speakers for any un-captured resources.
+  - Collected Zoom recordings from the Hosts/Co-Hosts of each session.
+  - Prepared videos for editing by
+   
+    - Organising files.
+    - Updating processing instructions for Events Team.
+      
+  - Followed up with speed-blog authors and connected them with the Communications Team for publication to the SSI blog.
+
+
+Month 2 (May 2020)
+^^^^^^^^^^^^^^^^^^^^^^
+
+Publicity
+  - Published a reflections blog post about `how we moved the event online in three weeks <https://software.ac.uk/blog/2020-05-18-cw20-how-move-event-online-three-weeks>`_.
+  
+Resources
+  - 9/12 discussion groups' `speed blogs published <https://software.ac.uk/tags/cw20-speed-blog-posts>`_.
+
+Month 3 (June 2020)
+^^^^^^^^^^^^^^^^^^^^^^
+
+Publicity
+  - Announced availability of `Infrastructure resources <https://software.ac.uk/news/collaborations-workshop-2020-resources-now-available>`_.
+  
+Resources
+  - 10/12 discussion groups' `speed blogs published <https://software.ac.uk/tags/cw20-speed-blog-posts>`_.
+  - Infrastructure resources sanitised and `shared <https://doi.org/10.6084/m9.figshare.c.5026400.v3>`_.
+
+
+Month 4 (July 2020)
+^^^^^^^^^^^^^^^^^^^^
+
+Publicity
+  - Announced availability of `session recordings <https://software.ac.uk/news/collaborations-workshop-2020-session-recordings-now-available>`_.
+  
+Resources
+  - 100% of videos processed and `uploaded <https://www.youtube.com/playlist?list=PLpX1jXuNTXGoVHrK6IPrn4Ymwaj6l4KLJ>`_.
+
+
+Month 5 (August 2020)
+^^^^^^^^^^^^^^^^^^^^
+
+- 11/12 discussion groups' `speed blogs published <https://software.ac.uk/tags/cw20-speed-blog-posts>`_ - the last one may be published in the future if the group members decide to.
+
```

Comment:
Do we want a Month 6 (September 2020)

+ EOG case study completed and reviewed